### PR TITLE
Fix a multicontract bug

### PR DESCRIPTION
**Pull request by norhh**
Fixes a case where 3rd nested call reverts due to incorrect calldata.

### CODE REVIEW

File: mythril/interfaces/cli.py [hunk 1]
```diff
@@ -563,7 +563,7 @@ def set_config(args: Namespace):
     ):
         config.set_api_from_config_path()
 
-    if args.__dict__.get("address", None):
+    if args.__dict__.get("rpc", None):
```
Comment (b-mueller) [inline mythril/interfaces/cli.py:566]: Why this change? We also need to establish an RPC connection to the default URL when there's no `--rpc` arg right?

File: mythril/laser/ethereum/call.py [hunk 2]
```diff
@@ -178,14 +178,6 @@ def get_call_data(
             else memory_size
         ),
     )
-
```
Comment (b-mueller) [inline mythril/laser/ethereum/call.py:181]: I'll just trust you that this fixes everything :) Can't replicate it right now because I'd have to context switch my whole test environment

File: mythril/interfaces/cli.py (hunk 1 above)
Comment (norhh) [reply mythril/interfaces/cli.py:566]: If rpc is not set then it takes the value from config at line 564. Without this we can't use testnets or ganache for queries when using solidity files.

File: mythril/laser/ethereum/call.py (hunk 2 above)
Comment (norhh) [reply mythril/laser/ethereum/call.py:181]: I am not sure yet. I didn't run it with the contract on atlassian. This is a bugfix for a tailor made contract which is a bit similar to original contract. But it should atleast fix one problem. Will test it then
